### ISSUE TITLE
Kernel Restarting: The Kernel appears to have died, it will restart automatically.

### ISSUE THREAD

**Issue by ayangupt90** [Vikramank/Deep-Learning-#6]
After solving previous problem a new problem has arrived. The problem is:
After completing 1st iteration any Browser is unable to show Total amount of accuracy depend on Testing & validation data set. For more clearance a screenshot has given:
![screenshot from 2018-06-18 16-59-45](https://user-images.githubusercontent.com/39652579/41819947-1756adf0-77e7-11e8-92aa-012090515e3f.png)
However, when i've tried to run the same code with 10,000 number of training data set it shows low amount of accuracy (average 50-55%) which is violate the main target of the project:
![screenshot from 2018-06-24 19-47-17](https://user-images.githubusercontent.com/39652579/41820009-34146fb2-77e8-11e8-9d65-85daa329c834.png)
By this, classifier cant do correct  prediction i.e. near of 50% images gives correct result & rest are not.... 

Actually my target point is to run complete program without any issue. But now after solving one software issue this issue depend on software or hardware issue .... i really cant understand :( !!

**Comment by Vikramank** [comment]
Hi,  extremely sorry for the delay in my response. As long as you the test accuracy is increasing (as in your case) you don't have to worry. Run for more epochs. It is likely getting hanged because you might have less RAM and since we are loading all the images in the test/validation set at the same time, the load in CPU RAM is likely to be high. To negate this, you can perhaps send a small batches of test/validation images
